src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -44,6 +49,14 @@ def test_u_nk(mbar_filename,
 
     assert u_nk.index.names == names
 
+@pytest.mark.parametrize("mbar_filename",
```

Comment:
I think we don't need two tests for this? Assume that the files are not that different?

Reply:
The file is the same, but the MBAR energies and the dHdl values are in different places, and are read a little bit differently. Unfortunately right now the parser parses the file two times to read the different values, hence I wrote the two tests. In the future implementation I want to develop, both values will be read loading the file a single time, so the tests will be "merged".